src/Libraries/Microsoft.Extensions.Http.Resilience/Resilience/Internal/ByAuthorityStrategyKeyProvider.cs

```diff
@@ -10,40 +10,40 @@
 
 namespace Microsoft.Extensions.Http.Resilience.Internal;
 
-internal sealed class ByAuthorityPipelineKeyProvider : IPipelineKeyProvider
+internal sealed class ByAuthorityStrategyKeyProvider : IStrategyKeyProvider
 {
     private readonly Redactor _redactor;
     private readonly ConcurrentDictionary<(string scheme, string host, int port), string> _cache = new();
 
-    public ByAuthorityPipelineKeyProvider(Redactor redactor, DataClassification _)
+    public ByAuthorityStrategyKeyProvider(Redactor redactor, DataClassification _)
```

Comment:
Who allocates this redactor?

If there is already a redactor, then this constructor doesn't need a DataClassification instance, so can we just remove it?

Reply:
True, removed.